src/main/java/com/novoda/notils/logger/simple/Log.java

```diff
@@ -206,6 +209,13 @@ public static void setShowLogs(boolean showLogs) {
      * @return {@code true} if logs are active, {@code false} if deactivated
      */
     public static boolean shouldShowLogs() {
-        return SHOW_LOGS;
+        if (INITIALISED) {
+            return SHOW_LOGS;
+        } else {
+            throw new DeveloperError("#rekt - To use simple logger you need to have called setShowLogs(boolean). "
```

Comment:
I like to have sub-system level granularity for logging, so that I can switch on logging for network, business logic and ui separately.   
But probably out of scope here.